tools/onnx_subgraph/CMakeLists.txt

```diff
@@ -13,3 +13,29 @@ find_package(Python3 COMPONENTS Interpreter Development REQUIRED)
 
 include_directories(${CMAKE_CURRENT_BINARY_DIR})
 include_directories(${Python3_INCLUDE_DIRS})
+
+set(ONNX_SUGRAPH_FILES                                                          
+    test_model_download.sh
+ )                                                                            
+                                                                              
+ foreach(ONNX_SUGRAPH IN ITEMS ${ONNX_SUGRAPH_FILES})                                                                                                        
```

Comment:
1 space is used for indentation.
it would be better to use 2/4/6 or 4/8/12 for indentation.